app/web/template/layout/page.gohtml

```diff
@@ -152,6 +152,18 @@
                     {{ tr .App "termsAndConditions" }}
                   </a>
                 </li>
+                <li class="govuk-footer__inline-list-item">
+                  <a class="govuk-footer__link" href="#5">
+                    {{ tr .App "termsAndConditions" }}
+                  </a>
+                </li>
+                {{ if eq false .App.IsProduction }}
+                  <li class="govuk-footer__inline-list-item">
+                    <a class="govuk-footer__link" href="{{ if containsStr "showTransKeys=1" .App.Query }}{{ .App.Lang.BuildUrl .App.Page }}{{ else }}{{ .App.Lang.BuildUrl .App.Page }}?showTransKeys=1{{ end }}">
```

Comment:
it would be nicer to have a `.App.ShowTranslationKeys` property instead of using the query